### PR TITLE
Boot5update

### PR DESCRIPTION
**Pull request by planetoftheweb**

### CODE REVIEW

**Comment by planetoftheweb** [comment]
I'm thinking about updating this to Bootstrap 5. Any thoughts?

**Comment by jojohumphreys** [comment]
Yeah, that's a great idea! I say let's do It!

File: docs/columns.html
Comment (jojohumphreys) [inline docs/columns.html:1]: This is looking pretty good.

File: docs/media.html [hunk 1]
```diff
@@ -93,10 +88,6 @@ <h4 class="">Media Three</h4>
       integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
```
Comment (jojohumphreys) [inline docs/media.html:88]: I'm noticing that you left a copy of jQuery here, but it's not necessary with Bootstrap 5

**Review by jojohumphreys — CHANGES_REQUESTED** [review]
Looks great, you may want to remove all of the references to jQuery since it's no longer necessary.

File: docs/media.html (hunk 1 above)
Comment (planetoftheweb) [reply docs/media.html:88]: Got it. I'll remember to remove that everywhere.

File: docs/columns.html
Comment (planetoftheweb) [reply docs/columns.html:1]: thanks

Comment (planetoftheweb) [reply docs/columns.html:1]: Yep, appreciate the thoughts

**Pull request closed without merging**